pkg/controller/controller.go

```diff
@@ -1049,6 +1051,24 @@ func (cont *AciController) Run(stopCh <-chan struct{}) {
 				cont.opflexDeviceDeleted(dn)
 			})
 
+		if !cont.config.DisableServiceVlanPreprovisioning && !strings.Contains(cont.config.Flavor, "openstack") {
+			if cont.config.AEP == "" {
+				cont.log.Error("AEP is missing in configuration, preprovisioning of service vlan will be disabled")
+			} else {
+				infraRtAttEntPFilter := fmt.Sprintf("and(wcard(infraRtAttEntP.dn,\"/attentp-%s/\"))", cont.config.AEP)
+				cont.apicConn.AddSubscriptionClass("infraRtAttEntP",
+					[]string{"infraRtAttEntP"}, infraRtAttEntPFilter)
+				cont.apicConn.SetSubscriptionHooks("infraRtAttEntP",
```

Comment:
can we add a log statement at info level here indicating that we are subscribing to this particular AEP (or will that already get logged in AddSubscriptionClass?